synapse/config/server.py

```diff
@@ -294,32 +355,28 @@ def read_config(self, config, **kwargs):
         )
 
         # Attempt to create an IPSet from the given ranges
-        try:
-            self.ip_range_blacklist = IPSet(ip_range_blacklist)
-        except Exception as e:
-            raise ConfigError("Invalid range(s) provided in ip_range_blacklist.") from e
+
         # Always blacklist 0.0.0.0, ::
-        self.ip_range_blacklist.update(["0.0.0.0", "::"])
+        self.ip_range_blacklist = generate_ip_set(
+            ip_range_blacklist, ["0.0.0.0", "::"], config_path=("ip_range_blacklist",)
```

Comment:
I wonder if these default IPs should be CIDRs?

Reply:
I think so? Probably?

Reply:
Updated 0.0.0.0 -> 0.0.0.0/8 which matches the default blacklist.